kfdefs/base/jupyterhub/notebook-images/s2i-spark3.0.1-minimal-notebook.yaml

```diff
@@ -18,3 +18,4 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/llasmith/s2i-spark-container:py36-spark3.0.1-hadoop3.2.0
```

Comment:
I was tested this image in the staging env and I found that the `ImageStream` name should be the same name as the image. So [here in line 11](https://github.com/operate-first/apps/blob/0c50a36d273140972d9bf069be1fa53018dd8ca4/kfdefs/base/jupyterhub/notebook-images/s2i-spark3.0.1-minimal-notebook.yaml#L11), the name should be `s2i-spark-container`

Reply:
If you don't do this, we see the following error:
`Back-off pulling image "spark3.0.1-xskipper:py36-spark3.0.1-hadoop3.2.0"`

Reply:
@vpavlin @LaVLas is this the desired behavior?

Reply:
No. I created that image as a test to see if I could recreate the images that @oshritf experienced when building docker locally.  I never tested this with an actual notebook build so it should not be used in an official deployment

We do have a spark version update on the ODH roadmap but this particular image would not be used for that work.

Reply:
My question was more about the need for ImageStream name and Image name to be the same for JH to use the correct image.